rootfs/api/models/release.py

```diff
@@ -134,7 +134,7 @@ def publish(self):
         deis_registry = bool(self.build.source_based)
         publish_release(source_image, self.image, deis_registry, self.get_registry_auth())
 
-    def get_port(self, routable=False):
```

Comment:
glad we're removing the `routable` flag. That always threw me off when making calls to it